README.md

```diff
@@ -1,62 +1,72 @@
-# My-Portfolio
+# 📖 [My-portfolio]
 
 [My-Portfolio] is project to show my main projects in the MIcroverse programme.
 
 ## 🛠 Built With
-Client: [React], [Redux]
-Server: [Node.js]
-Database [MongoDB]
+The following technologies has been used in developping this project. 
 
-## Key Features
+### Tech Stack
+
+```[Tech-stack]
+client: React.js
+Server: Express.js
+database: PostgreSQL
+```
+
+### Key Features
 
 [My-Portfolio] key features are the following main projects:
 
 ``
-[Project-1](Setup and mobile version skeleton)
-[Project-2](Mobile version)
-[Project-3](Contact form)
-[Project-4](Desktop version)
-[Project-5](Application deploy)
-[Project-6](Evaluate accessibility)
-[Project-7](Mobile menu)
-[Project-8](Details modal)
-[Project-9](Validate contact form)
-[Project-10](Preserve data in the browser)
+#### [Project-1](Setup and mobile version skeleton)
+#### [Project-2](Mobile version)
+#### [Project-3](Contact form)
+#### [Project-4](Desktop version)
+#### [Project-5](Application deploy)
+#### [Project-6](Evaluate accessibility)
+#### [Project-7](Mobile menu)
+#### [Project-8](Details modal)
+#### [Project-9](Validate contact form)
+#### [Project-10](Preserve data in the browser)
 ``
 
 
-## 💻 Getting Started
-To get a local copy up and running, follow these steps.
+## 🚀 Live Demo
+
+Live Demo Link
+(back to top)
+
+# 💻 Getting Started
+
+## Setup
+Clone this repository to your desired folder:
+https://github.com/diarisdiakite/[........] 
 
-### Prerequisites
+## Prerequisites
 In order to run the projects on this portfolio you need:
 Have some knowledge of HTML and CSS
 Be familiar with git commands
 Basics understanding of Linters
 
-### Setup
-Clone this repository to your desired folder:
-https://github.com/diarisdiakite/[........] 
-
-### Install
+## Install
 Install the portfolio project with: [npm](https://www.npmjs.com/)
 
 ```[npm]
 npm install (--npm-command-project-name/app)
 ```
 
-### Usage
+## Usage
 To run the portfolio projects, execute the following command: Open the project in code editor
 
-### Run tests
+## Run tests
 To run tests, run the following command: Open the project in code editor and run the tests
 
-### Deployment
+## Deployment
 You can deploy the portfolio projects using: google cloud, Microsoft Azure, Netlify by giving credit on usign our template or contricute via pull requests (read more in the contributing section).
 
 (back to top)
 
-## 👥 Authors
+# 👥 Authors
 
 👤 Didy
 
```

Comment:
- [ ] [OPTIOANL] I kindly suggest adding pair-programming partners including social media handles to the authors' section. 😃

Reply:
Done @braincee.
However, I had already added the name of one collaborator in the Acknowledgement. Do they really need to be Authors to in my portfolio repo?